jep/1/README.adoc

```diff
@@ -0,0 +1,593 @@
+  rationale before writing code, the principle of "rough consensus
+  and running code" is still useful when it comes to resolving many
+  discussions of API details.
+. **References** -- When moving a JEP from a Draft to Accepted or Final state,
+  the references section should be updated to include links to the pull requests
+  and mailing list discussions which were involved in the process. The JEP
+  should self-document the process in which it was developed.
+
+The final implementation must include test code and documentation
+appropriate for either the Jenkins user or developer documentation.
+
+
+==== JEP Formats and Templates
+
+JEPs are UTF-8 encoded text files using the
+link:https://asciidoctor.org[AsciiDoc] format.  AsciiDoc allows for rich markup
+that is still quite easy to read, but also results in good-looking and
+functional HTML.
+
+
+==== JEP Header Preamble
+
+Each JEP must begin with an AsciiDoc table containing metadata relevant to the
+JEP:
+
+[source,asciidoc]
+----
+.Metadata
+[cols="2"]
+|===
+| JEP
+| 1
+
+| Title
+| Jenkins Enhancement Proposal Format
+
+| Author
+| link:https://github.com/rtyler[R Tyler Croy]
+
+| Status
+| :speech_balloon: Draft
+
+| Type
+| Process
+
+| Created
+| 2017-09-12
+|===
+----
+
+
+. **JEP** -- Proposal number, given by the JEP editors. Use "9999" until one is assigned.
+. **Title** -- Brief title explaining the proposal in fewer than 50 characters
+. **Author** -- Author/champion of the JEP, in essence, the individual
+  responsible for seeing the JEP through the process.
+. **Status** -- Draft :speech_balloon:, Deferred :hourglass:, Accepted :ok_hand:, Rejected :no_entry:, Withdrawn :hand:, Final :lock:, Replaced :dagger:, Active :smile:.
+. **Type** -- Describes the type of JEP: Standards, Informational, Process
+. **Created** -- Date (`%Y%m%d`) when the document was first created.
+
+
+
+A **BDFL-Delegate** row is used to record cases where the final decision to
+approve or reject a JEP rests with someone other than the BDFL.
+
+For a JEP where final pronouncement will be made on a list other than
+jenkinsci-dev@, a **Discussions-To** row will indicate the mailing list
+or URL where the pronouncement will occur. A temporary Discussions-To header
+may also be used when a draft JEP is being discussed prior to submission for
+pronouncement.
+
+JEPs may have a **Requires** row, indicating the JEP numbers that this
+JEP depends on.
+
+JEPs may also have a **Superseded-By** row indicating that a JEP has been
+rendered obsolete by a later document; the value is the number of the JEP that
+replaces the current document. The newer JEP must have a **Replaces** row
+containing the number of the JEP that it rendered obsolete.
+
+
+==== Auxiliary Files
+
+JEPs may include auxiliary files such as diagrams. Such files must be
+named appropriately, with lowercase letters and no spaces, and be included in
+the directory with the `README.adoc` describing the JEP.
```

Comment:
Having myself pushed big garbage in the past, I feel something like "a particular attention to the reasonable size of the resources and resolution will be made".
IOW, a diagram that could be readable in a 5x10cm rectangle *must* not be 12 MB.
Because this will impact the size and the easiness of cloning it then forever.

Reply:
@batmat  Could you submit a PR that includes you suggested changes?  What balance would you like to strike?

Reply:
@batmat  I believe tyler addressed this with a later change.  Please check and comment "RESOLVED" if resolved.